### ISSUE TITLE
Trailer showing black screen

### ISSUE THREAD

**Issue by DROIDshield** [jurialmunkey/skin.arctic.fuse#529]
### Skin section

Other

### Current Behavior

@jurialmunkey love the small screen for trailer playback but for some reason it only shows black on my shield but works fine on the laptop setup.

Both running latest skin and Kodi 20.3 all settings are the same on the shield and laptop but cannot get that screen to run any media. If I back out to full screen tailer that works as normal. 

Any ideas?
![20240303_035624](https://github.com/jurialmunkey/skin.arctic.fuse/assets/97023369/5423f4ba-5eed-43ca-a0eb-02f04c5b4004)

### Expected Behavior

Should play trailer in small screen when activated. 

### Steps To Reproduce

Play back trailer

### Screenshots and Additional Info

See image above.

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Try removing the animation here and restart Kodi and see if it helps.
https://github.com/jurialmunkey/skin.arctic.fuse/blob/01ce4ed211b3e57a09ffff3dbf71548eb2994694/1080i/Includes_Dialogs.xml#L735-L737

Also does the trailer dialog work for you? Because if it doesn't either then possibly system doesn't support overlay render.

**Comment by DROIDshield** [comment]
Tried removing the lines restarting and in dialog as well and no its still black screen.

As regards the shield pro if running PIP modes is the same thing as what you call overlay render then it does support this as various apps I use can reduce to PIP see pic and let me know if I've got it right or wrong..

![20240303_045045](https://github.com/jurialmunkey/skin.arctic.fuse/assets/97023369/49a9236b-ce27-41b2-9247-3e2bc23030fd)

**Comment by jurialmunkey** [comment]
It doesn't matter much if Android can do it in other apps, only if it is possible inside Kodi.

Can you let me know if the trailer dialog window works when background video is set?

1. Enable "Skin Settings > Appearance > Background video"
2. Enable "Skin Settings > Behaviour > Play trailers from information in > Dialog"
3. Go to Info dialog for a movie and click Trailer button to see if trailer plays in the pop-up dialog

**Comment by DROIDshield** [comment]
Made all changes as requested above and background video and dialogue work, see pics added. Then updated to the latest skin as you made a change to PIP this works but only after opening full screen I have to press the back key on remote once then the small screen shows with the trailer playing. 

I've gone back to just using background trailer for the moment as that works as always did.

The shield is a great toy but 3gig of ram is starting to freak me out as it just cannot handle running kodi skinned without a bit of glitching and crashing from time to time unlike my laptop which is a power house and runs with ease and very snappy so im not worried about this setting. I'll carry on without it.
![20240303_151814](https://github.com/jurialmunkey/skin.arctic.fuse/assets/97023369/3b6248fd-feeb-439a-b14c-18cb45033651)
![20240303_152014](https://github.com/jurialmunkey/skin.arctic.fuse/assets/97023369/bc31420e-cd94-414f-bd42-8108f8962e5e)
BTW I sent you a DM on the Kodi sight a few day's ago, not sure if you got it?

**Comment by jurialmunkey** [comment]
Hmm really strange that it works in the trailer dialog but not the other one. Has me a bit stumped as to the reason but the videowindow overlay has always been a bit finicky on some systems -- that's why I've got the background option there because I know it works.

Anyway if you're happy with background video then I'll close and I'm not really sure the issue and it's a bit of stumbling around in the dark without an Android device to test on.

>  BTW I sent you a DM on the Kodi sight a few day's ago, not sure if you got it?

Sorry I don't really bother checking my notifications on the forums because I get so many from subscribed threads, so I almost never see DMs there. I'll have a look now.
